content/docs/command-reference/plots/modify.md

```diff
@@ -0,0 +1,132 @@
+# plots modify
+
+Modify display properties of [plot metrics](/doc/command-reference/plots) files.
+
+## Synopsis
+
+```usage
+usage: dvc plots modify [-h] [-q | -v] [-t <path>] [-x X] [-y Y]
```

Comment:
```suggestion
usage: dvc plots modify [-h] [-q | -v] [-t <path>] [-x X] [-y Y]
```

`-t` actually accepts path or a standard template name (from .dvc/plots). Not sure if I would set the metavar to `<path>`. Better to keep it as TEMPLATE or whatever.

Reply:
I thought we had recently agreed to only use one style, yes (cc @shcheklein). I've opened issues/PRs in this in the core repo and tried to keep an eye on core PRs that introduce new metavars but it's hard to keep up.

We started changing this style because `OPTION_NAME` is "loud" and not descriptive. `<option_type>` gives some useful information (what kind of argument is accepted?) and is more similar to Git docs AFIAR.

Let's decide and apply throughout? Will probably need a separate issue though.

Reply:
> -t actually accepts path or a standard template name

Good point in this case though. Maybe just a more general `<text>` or `<name>`, or even `<name_or_path>` ?

Reply:
I think we can make it <template> in this case and just clarify that it can be a name or path

Reply:
hmm ... to me <template> is more explicit (explains the `-t` clearly) ... `text` - I don't know what does it mean, name is good by not precise, name_or_path - good but ugly :)

Reply:
<text> seems a bit ambiguous to me too, but I don't have a very strong opinion here. Personally I don't even read metavars that deeply when I look into commands help, I just use them to indicate to me whether or not this option expects a value and, if so, how many. I jump to description for more details. But I'm fine with a mixup for now, I guess we will evolve and will see what works best. Just would like these to be consistent in the future.

<name_or_path> is very good, I would totally use that if we are to change metavars.

Reply:
> I think we can make it <template> in this case

Updated to this in 0bfb2ad1. Cc @efiop 

But please lmk if we should rather do <name_or_path>. I would personally prefer that but no strong opinion.

Reply:
`name_or_path` sounds good for me

Reply:
Just noticed 

> `<name_or_path>` is very good, I would totally use that if we are to change metavars.

OK, done that now.

> Personally I don't even read metavars that deeply when I look into commands help, I just use them to indicate to me whether or not this option expects a value and, if so, how many

Good points. But if we can even convey some idea on what kind or argument is accepted, why not? 🙂